lib/phoenix.ex

```diff
@@ -10,6 +10,8 @@ defmodule Phoenix do
     * [Phoenix](https://hexdocs.pm/phoenix) - the Phoenix web framework
       (these docs)
 
+    * [Phoenix.js](/js) - Phoenix Channels JavaScript client
+
```

Comment:
I don't think this link will work on hexdocs.pm since it will link to `https://hexdocs.pm/js`. I think you need the full absolute link `https://hexdocs.pm/phoenix/js`, this will unfortunately not be versioned but I don't know how to fix that.